react/week1/my-app/src/components/todo.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Todo = ({ todo }) => {
+  return (
+    <div className="todoWrapper">
+      {todo.title}, {todo.deadline}
+    </div>
```

Comment:
It is semantically better to use the html list element for lists(`<ul>` for unordered list and `<ol>` for ordered list).